rust/arrow-flight/Cargo.toml

```diff
@@ -27,7 +27,10 @@ license = "Apache-2.0"
 
 [dependencies]
 arrow = { path = "../arrow", version = "3.0.0-SNAPSHOT" }
-tonic = "0.3"
+tonic = "0.3.1"
```

Comment:
I wonder if there is some reason to upgrade tonic in this same PR?

Reply:
I had compilation failures otherwise, but it's strange that other PRs do not seem to have the same issue. I'll check again and maybe move this to a separate PR.

Reply:
Removed the dependency again and CI still works, not sure what the issue was locally.